Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -65,11 +65,13 @@ public final class GestureManager: NSObject {
     /// Set this delegate to be called back if a gesture begins
     public weak var delegate: GestureManagerDelegate?
 
+    /// Internal delegate for gesture recognizers
+    internal weak var gestureRecognizerDelegate: GestureRecognizerDelegate?
```

Comment:
Can this be a `let` reference to a non-optional delegate instead of a `weak var`?

Reply:
I pushed a change to address this but it means silencing a swift lint warning